pkgs/development/libraries/abseil-cpp/202301.nix

```diff
@@ -18,13 +19,17 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   cmakeFlags = [
+    "-DABSL_BUILD_TEST_HELPERS=ON"
+    "-DABSL_USE_EXTERNAL_GOOGLETEST=ON"
     "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}"
   ] ++ lib.optionals (cxxStandard != null) [
     "-DCMAKE_CXX_STANDARD=${cxxStandard}"
   ];
 
   nativeBuildInputs = [ cmake ];
 
+  buildInputs = [ gtest ];
```

Comment:
just curious, can gtest be in `nativeBuildInputs`?

Reply:
I double checked with `strictDeps` and it cannot be native.

Reply:
I updated the PR to set `strictDeps = true`.